Tests/Support/Snapshots/OctoKit/Sources/Entities/Repository.swift

```diff
@@ -169,25 +169,25 @@ public struct Repository: Codable {
     /// Whether to allow rebase merges for pull requests.
     ///
     /// Example: true
-    public var allowRebaseMerge: Bool
+    public var allowRebaseMerge: Bool?
     public var templateRepository: TemplateRepository?
     public var tempCloneToken: String?
     /// Whether to allow squash merges for pull requests.
     ///
     /// Example: true
-    public var allowSquashMerge: Bool
+    public var allowSquashMerge: Bool?
     /// Whether to allow Auto-merge to be used on pull requests.
     ///
     /// Example: false
-    public var allowAutoMerge: Bool
+    public var allowAutoMerge: Bool?
```

Comment:
Are these changes right? I'm not sure what has caused them but we don't seem t have modified the schema have we?

Reply:
My understanding of the OpenAPI spec, limited as it may be, is that it is technically more accurate because these properties are not required in their respective definitions.

Reply:
I'm trying to unwind my changes since it's been a few months since I did most of this work, in order to give a definitive answer on this. It might be possible to split this out, but it's not immediately clear.